src/Sokoban.hs

```diff
@@ -1,214 +1,234 @@
--- Data types, world rules
+-- The main file for the underlying logic of the game.
 module Sokoban
-(
-    GameObject(..),
-    Background(..),
-    Cell(..),
-    Environment,
-    Movement(..),
-    move,
-    sampleLevel,
-    emptyCell,
-    playerCell,
-    wallCell,
-    trashCell,
-    isValidMove
-)
-    where
+  ( GameObject(..)
+  , Background(..)
+  , Cell(..)
+  , Environment
+  , Movement(..)
+  , MoveStatus(..)
+  , move
+  , sampleLevel
+  , emptyCell
+  , playerCell
+  , wallCell
+  , trashCell
+  , stashCell
+  , isValidMove
+  ) where
```

Comment:
Not sure if we need all of these to be exported. It doesn't change the behavior so its not critical to remove excess exports